charts/kubernetes-agent/Chart.lock

```diff
@@ -0,0 +1,6 @@
+dependencies:
+- name: kubernetes-monitor-chart
+  repository: oci://docker.io/octopusdeploy
```

Comment:
Is it possible to change the repo this chart is pulled from? (say if I want to pull from Artifactory?)

Reply:
kk, we can always change it if we need to in the future 👍